importers/file_meta.py

```diff
@@ -18,10 +41,16 @@ def _do_import(self, doc, filepath):
     headers = {
       'Cookie': self.cookie
     }
-    res = requests.post(self.url, files=files, headers=headers, data={
-      key:d['value']
-      for key, d in doc.items() if key and d
-    }, verify=False)
+
+    # Strip out empty fields (??)
+    mdata = {key: d for key, d in doc.items() if key and d}
+
+    if self.verbose and not self.quiet:
```

Comment:
I think this should be `if not self.quiet:`. The `verbose` flag is about whether or not logs get printed to stdout.

Reply:
I guess I'm feeling confused about this. I know you're super busy and don't want to waste your time on minutiae.

It should be possible to:
- Print logs to stdout and generate no logfiles
- Print logs to stdout and generate logfile based on `_log` method
- Print logs to stdout, generate logfile based on `_log` method, and generate JSON metadata logs
- Don't print logs and generate logfile based on `_log` method
- Don't print logs, generate logfile based on `_log` method, and don't generate JSON metadata logs

Right now I don't think all of these combos are possible. The `verbose` flag should only every be used to designate whether or not logs are printed to console, and is totally indep. of whether or not a logfile is set. I think what's changed here is before if `logfile` wasn't set then no logfiles were created. Now that there's a default value it isn't possible to use that parameter the same way. Maybe we should remove the option of having a custom logfile name and just have a `--log_to_file` flag. And then maybe we need to add a `--log_meta` flag just to handle metadata logging.

I can make these changes during merge.

Reply:
Yes, you are quite right here. I changed it and pushed.

Reply:
So I changed the conditional above to: `if not self.quiet:` As for the longer issue of opening all possible logging options, you are also right about that. If you could implement upon merge, that would be a big help for me. Your suggestion sounds good.